src/poetry/console/commands/version.py

```diff
@@ -29,7 +29,10 @@ class VersionCommand(Command):
             optional=True,
         )
     ]
-    options = [option("short", "s", "Output the version number only")]
+    options = [
+        option("short", "s", "Output the version number only"),
+        option("dry-run", None, "Outputs the operations but will not execute anything"),
```

Comment:
Maybe reword this so it is specific to the command and indicate that we do not write aything to the toml file.

Reply:
Updated